CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## [2.0.0] - 2024-12-12
+
+- Drop usage of System.Data.SqlClient in favor of Microsoft.Data.SqlClient
+- Bump target framework from net451 to net462
+- Server certificate parameter in connection string is required now
```

Comment:
It is not required, but the trust cetificate is not on by default in the new driver. So we should mention that if someone is upgrading then self signed certificates are not accepted by default anymore.